README.md

```diff
@@ -661,6 +666,17 @@ Provides access to internal task counters.
 var counters = sharp.counters(); // { queue: 2, process: 4 }
 ```
 
+#### Experimental: sharp.compare(filename1, filename2, callback)
+
+`filename1` and `filename2` are strings containing the filenames of the images to compare.
+
+`callback` is called with two arguments `(err, info)` where:
+
+* `err` contains an error message, if any.
+* `info` contains the info about the difference between the two images such as `isEqual` (Boolean), `meanSquaredError` (Number; present iff `status='success'`, otherwise `undefined`), and `status` (String; one of `success`, `mismatchedDimensions`, `mismatchedBands`, `mismatchedType`).
+
```

Comment:
Added `status` to not report certain errors as fatal, e.g. mismatched dimensions, bands, etc.